test/budgie_web/live/budget_list_live_test.exs

```diff
@@ -57,17 +57,18 @@ defmodule BudgieWeb.BudgetListLiveTest do
       conn = log_in_user(conn, user)
       {:ok, lv, _html} = live(conn, ~p"/budgets/new")
 
-      form = form(lv, "#create-budget-modal form")
-
-      {:ok, _lv, html} =
-        render_submit(form, %{
+      form =
+        form(lv, "#create-budget-modal form", %{
```

Comment:
Putting the parameters into `form/3` validates that the inputs exist on the page.

See [this doc](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveViewTest.html#form/3) for details.